openssl/src/ssl/mod.rs

```diff
@@ -339,7 +373,13 @@ impl SslContext {
             return Err(SslError::get());
         }
 
-        Ok(SslContext { ctx: ctx })
+        let ctx = SslContext { ctx: ctx };
+
+        if method.is_dtls() {
+            ctx.set_read_ahead(1);
```

Comment:
You are correct about this manpage, but the manpage is incorrect ;)
I think the scenario in the manpage is when using a UDP socket directly (without a MemBIO). The call to `set_read_ahead()` is definitely required in the context where we use MemBIO.